### ISSUE TITLE
publish.html does not exist

### ISSUE THREAD

**Issue by tyeeman** [cesarparent/Asteroid#1]
Trying to install this. "publish\index.php" on line 52 says use publish.html, but it does not exist anywhere!

**Comment by cesarparent** [comment]
Indeed, I will look into the repo and add it. That being said, this is (very) old and badly-made code, I would be careful about using it in production. The successor I made after (saturn, <https://github.com/cesarparent/saturn>) is probably safer.

**Comment by tyeeman** [comment]
I'm trying saturn now but getting a lot of undefined variables errors.

**Comment by cesarparent** [comment]
mmh. Can you open an issue there with details (screenshots/error messages)? That way I can have a better look at what's wrong.
